roles/common/templates/uug_ansible_wrapper.py

```diff
@@ -138,18 +138,17 @@ def __init__(self):
         contents = Gtk.Box(orientation=Gtk.Orientation.VERTICAL, spacing=12)
         contents.set_border_width(10)
 
-        label = Gtk.Label(
-            "Select the course configurations to add/update"
-            " (at this time courses cannot be removed)."
-        )
+        label = Gtk.Label(label=
+                          "Select the course configurations to add/update"
+                          " (at this time courses cannot be removed).")
```

Comment:
This is unlikely to be syntactically valid. You'll need to wrap the strings in `()` so that Python actually knows they're part of the same expression. Alternatively, we'll need to just accept really long lines in some places. Or move to a better method of string management.

```suggestion
        label = Gtk.Label(
            label=(
                "Select the course configurations to add/update"
                " (at this time courses cannot be removed)."
            )
        )
```

Reply:
For clarification, this only applies to strings that span multiple lines.